cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
Will split out the instanceState config in a more general way, and then move the hostfolders first to the all 'global' scope... after this I can implement the flag `--instance-only` for `Add` of a hostfolder. This way, if time is limited, at least the global (Hardy's preferred way ;-) is implemented, while `--instance-only` can be implemented as a possible increment is not possible due to time constraints.

Just need to see how we can guarantee that names remain unique among 'global' and 'instance'-level, without over-complicating things.